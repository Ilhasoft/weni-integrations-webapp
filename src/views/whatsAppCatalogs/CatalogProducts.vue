<template>
  <ProductList class="whatsapp-catalog-products" :catalogName="catalogName" />
</template>

<script>
  import ProductList from '@/components/whatsAppCatalogs/ProductList';

  export default {
    name: 'CatalogProducts',
    props: {
      catalogName: {
        type: String,
        required: true,
      },
    },
    components: {
      ProductList,
    },
  };
</script>

<styles scoped lang="scss">
.whatsapp-catalog-products {
  display: flex;
  overflow: hidden;
  height: 100%;
}
</styles>
