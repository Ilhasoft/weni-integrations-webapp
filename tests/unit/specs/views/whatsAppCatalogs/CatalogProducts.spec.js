import { shallowMount, createLocalVue } from '@vue/test-utils';
import CatalogProducts from '@/views/whatsAppCatalogs/CatalogProducts.vue';

const localVue = createLocalVue();

describe('views/whatsAppCatalogs/CatalogProducts.vue', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallowMount(CatalogProducts, {
      localVue,
      propsData: {
        catalogName: 'catalog name',
      },
    });
  });

  it('should be rendered properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
