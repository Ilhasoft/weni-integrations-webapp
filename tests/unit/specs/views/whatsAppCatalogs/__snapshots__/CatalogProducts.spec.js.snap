// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/whatsAppCatalogs/CatalogProducts.vue should be rendered properly 1`] = `<productlist-stub catalogname="catalog name" class="whatsapp-catalog-products"></productlist-stub>`;
